database/dbPointsProg.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
```

Comment:
good formatting and indention. don't forget the header comment as well as variable name consistancy.